src/DetailsView/components/left-nav/assessment-left-nav.tsx

```diff
@@ -33,6 +34,20 @@ export type AssessmentLeftNavLink = {
     status: ManualTestStatus;
 } & BaseLeftNavLink;
 
+export type TestGettingStartedNavLink = {
+    testType: VisualizationType;
+} & BaseLeftNavLink;
+
+export type TestRequirementLeftNavLink = {
+    displayedIndex: string;
+    testType: VisualizationType;
+} & AssessmentLeftNavLink;
+
+export type onTestRequirementClick = (
+    event: React.MouseEvent<HTMLElement>,
+    item: TestRequirementLeftNavLink,
+) => void;
+
```

Comment:
I'm curious why we don't have an `onTestGettingStartedClick` type here as well?

Reply:
I'll likely add it in the next PR when I'll need it.